modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -737,26 +737,47 @@ private boolean _isActive(ObjectField objectField) throws PortalException {
 		return true;
 	}
 
+	private void _removeTimeFromDateString(
+		DDMFormField ddmFormField, Map<String, Serializable> values) {
+
+		for (Map.Entry<String, Serializable> entry : values.entrySet()) {
+			if (!Objects.equals(entry.getKey(), ddmFormField.getName())) {
+				continue;
+			}
+
+			String dateString = String.valueOf(entry.getValue());
+
+			values.replace(
+				entry.getKey(), entry.getValue(),
+				dateString.replaceAll(
+					" [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}.[0-9]", ""));
+		}
+	}
+
 	private void _setDateDDMFormFieldValue(
 		List<DDMFormField> ddmFormFields, Map<String, Serializable> values) {
 
 		for (DDMFormField ddmFormField : ddmFormFields) {
-			if (!StringUtil.equals(ddmFormField.getType(), "date")) {
-				continue;
-			}
+			if (StringUtil.equals(ddmFormField.getType(), "fieldset")) {
+				List<DDMFormField> nestedDDMFormFields =
+					ddmFormField.getNestedDDMFormFields();
 
-			for (Map.Entry<String, Serializable> entry : values.entrySet()) {
-				if (!Objects.equals(entry.getKey(), ddmFormField.getName())) {
-					continue;
-				}
+				for (DDMFormField nestedDDMFormField : nestedDDMFormFields) {
+					if (!StringUtil.equals(
+							nestedDDMFormField.getType(), "date")) {
 
-				String dateString = String.valueOf(entry.getValue());
+						continue;
+					}
 
-				values.replace(
-					entry.getKey(), entry.getValue(),
-					dateString.replaceAll(
-						" [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}.[0-9]", ""));
+					_removeTimeFromDateString(nestedDDMFormField, values);
+				}
 			}
+
+			if (!StringUtil.equals(ddmFormField.getType(), "date")) {
+				continue;
+			}
+
+			_removeTimeFromDateString(ddmFormField, values);
```

Comment:
@mateussgn @marcelabc I think we can turn this method into a recursive method. Can any of you two help @danvb112 with this?

If you need more details, just call me :D